src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Core/IdeaCollection/Process/Process.ts

```diff
@@ -74,6 +74,7 @@ export var detailDirective = (
                         }
 
                         scope.contentType = scope.processProperties.proposalVersionClass.content_type;
+
```

Comment:
This is a placeholder commit (the merge commit would have been empty otherwise).